roles/zabbix_server/tasks/main.yml

```diff
@@ -27,8 +27,20 @@
   tags:
     - always
 
-- name: Install the correct repository
-  ansible.builtin.include_tasks: "{{ ansible_os_family }}.yml"
+- name: Install Repository
+  ansible.builtin.include_role:
+    name: community.zabbix.zabbix_repo
+  vars:
+    zabbix_repo_apt_priority: "{{ zabbix_server_apt_priority | default (omit) }}"
+    zabbix_repo_version: "{{ zabbix_server_version }}"
+    zabbix_repo_package: "zabbix_server-{{ zabbix_proxy_database }}"
+  when: zabbix_manage_repo | default(true)
+
+- name: "Configure SELinux when enabled"
+  ansible.builtin.include_tasks: selinux.yml
+  when:
+    - zabbix_server_selinux | default (false) | bool
```

Comment:
You sure we don't want to reuse the previous `when: ansible_facts.selinux.status | default('disabled') == 'enabled'` here? i.e. anything that reports selinux_enabled we apply on. (Suse also comes with selinux, #1194 and I think he would have an easier job (smaller PR) adding support after this merges.)

Reply:
Hmm, actually I think even better would be to put this when condition into defaults, so users can explicitly override us. Shame I didn't think about that in the selinux refactor.

```
--- # defaults/main.yml
zabbix_server_selinux: "{{ ansible_facts.selinux.status | default('disabled') == 'enabled' }}"
```

Reply:
Don't remember where we had that but now that you say it ya I remember seeing that.  Ya I think I like that better.